gensim/parsing/preprocessing.py

```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
 
 
 def strip_tags(s):
+    """Takes string and removes tags
+    Parameters
+    ----------
+    s : string
+    Returns
+    -------
+    string
+        Unicode string without tags
+
+
+    Examples
+    --------
+    >>from gensim.parsing.preprocessing import strip_tags
```

Comment:
`>>from gensim.parsing.preprocessing import strip_tags` -> `>>> from gensim.parsing.preprocessing import strip_tags`, here and anywhere.